src/blocks/navigation/style.scss

```
@@ -268,7 +270,7 @@ $link-color-property: var(
 			@include parent-toggles-menus-styles();
 		}
 	}
-	//Mobiel parent toggle sub menus
+	//Mobile parent toggle sub menus
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)